### PR TITLE
feat: Items commands 

### PR DESCRIPTION
**Pull request by Prakkkmak**

### CODE REVIEW

File: code/Modules/Commands/ItemsCommands.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+using ssl.Player;
+
+namespace ssl.Modules.Commands
+{
+    public class ItemsCommands
+    {
+        /// <summary>
+        /// Clear the inventory
+        /// </summary>
+        [AdminCmd("clear")]
```
Comment (Nowp) [inline code/Modules/Commands/ItemsCommands.cs:13]:
```suggestion
        [AdminCmd("inv_clear")]
```

**Review by Nowp — COMMENTED** [review]
Is AdminCmd executing client side or server side ?

File: code/Modules/Commands/ItemsCommands.cs (hunk 1 above)
Comment (Prakkkmak) [reply code/Modules/Commands/ItemsCommands.cs:13]: I wanted to make easy commands for fast tasks. In mc it is clear

Comment (Nowp) [reply code/Modules/Commands/ItemsCommands.cs:13]: yeah but won't it overwrite a "clear" command to clear the console ?

Comment (Nowp) [reply code/Modules/Commands/ItemsCommands.cs:13]: I checked and there's already "clear" so we'll go for the prefix "inv"

**Review by Nowp — APPROVED** [review]

**Pull request merged**